### DISCUSSION TITLE
[Question] Is it possible to know if the form is valid?

### DISCUSSION THREAD

**Discussion by erossini** [Blazored/FluentValidation#90] [Q&A]
In my project I have a form repeated a lot of time. If one of them has an issue, I want to display a custom message to the user at the top of the form. How can I know if the form is in error?

Thank you in advance,
Enrico

**Comment by chrissainty** [comment]
Do you want to only display one error per form and not field level error messages? If so you would need to write some custom code for doing that. This library will only output field level error messages. I would look into the EditContext. You can use this to find out if there are any error messages and then base you logic on that. It is covered in the official docs, I beleive.
